index.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.contact-form {
+  max-width: 800px;
+}
+
+.contact-form .cf-item {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 1.6rem;
+}
+
+.contact-form .cf-input {
```

Comment:
I'd make some of these inputs wider so they're more likely to match the size of what the content will be. E.g. I don't want to have to scroll across in the input to check that I've typed my email address correctly.